### PR TITLE
Add Azure publish support

### PR DESCRIPTION
**Pull request by bowdenk7**
This PR is primarily for adding guidance on publishing the app. Notable changes:
- Added new sections to the README
- Replaced logger with Winston
- Updated how env vars and secrets are managed

### CODE REVIEW

**Comment by bowdenk7** [comment]
image
 Pasting this here for hosting purposes...

**Comment by bowdenk7** [comment]
@peterblazejewicz any chance you also want to take a look?

File: src/util/logger.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import * as winston from "winston";
```
Comment (peterblazejewicz) [inline src/util/logger.ts]: The `esModuleInterop: true` has been added to simplify this (and help users)
```js
import winston from "winston"`
```
:)

File: src/util/secrets.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import logger from "./logger";
+import * as dotenv from "dotenv";
+import * as  fs from "fs";
```
Comment (peterblazejewicz) [inline src/util/secrets.ts]: Same as above, the `esModuleInterop` is enabled in project to ease migration to TS :)

**Pull request merged**